src/newcharacter.cpp

```diff
@@ -2293,11 +2296,19 @@ tab_direction set_description(WINDOW *w, player *u, const bool allow_reroll, poi
         const std::string action = ctxt.handle_input();
 
         if (action == "NEXT_TAB") {
-            if( OPTIONS["POINT_DISTRIBUTION"] != "freeform" && !points.is_valid() ) {
-                popup(_("Too many points allocated, change some features and try again."));
+            if (OPTIONS["POINT_DISTRIBUTION"] != "freeform" && !points.is_valid() ) {
+                if (points.skill_points_left() < 0 ) {
+                        popup(_("Too many points allocated, change some features and try again."));
```

Comment:
If I do this, then I have to differentiate between the error during Multiple Pools mode, and the error during Single Pool. 
On a related not, the only way to have skill points cause an error is to have a negative sum of points. I can probably just get rid of that error check altogether. Thoughts?